aaelf64-morello/aaelf64-morello.rst

```diff
@@ -0,0 +1,825 @@
+..
+   Copyright (c) 2011-2020, Arm Limited and its affiliates.  All rights reserved.
+   CC-BY-SA-4.0 AND Apache-Patent-License
+   See LICENSE file for details
+
+.. |release| replace:: 2020Q3
+.. |date-of-issue| replace:: 1\ :sup:`st` October 2020
+.. |copyright-date| replace:: 2011-2020
+.. |footer| replace:: Copyright © |copyright-date|, Arm Limited and its
+                      affiliates. All rights reserved.
+
+.. _AAELF64: https://github.com/ARM-software/abi-aa/releases
+.. _MORELLO_ARM: https://developer.arm.com/documentation/ddi0606/latest
+.. |tlsdesc-url| replace:: http://www.fsfla.org/~lxoliva/writeups/TLS/paper-lk2006.pdf
+.. _TLSDESC: http://www.fsfla.org/~lxoliva/writeups/TLS/paper-lk2006.pdf
+
+Morello extensions to ELF for the Arm\ :sup:`®` 64-bit Architecture (AArch64)
+*****************************************************************************
+
+.. class:: version
+
+|release|
+
+.. class:: issued
+
+Date of Issue: |date-of-issue|
+
+.. class:: logo
+
+.. image:: ../Arm_logo_blue_150MN.png
+
+.. section-numbering::
+
+.. raw:: pdf
+
+   PageBreak oneColumn
+
+Change control
```

Comment:
Could you keep the order of the initial sections in line with the documents already up on github, with the license text etc ahead of change control. And the preface on top, renamed as preamble, etc..